libminifi/test/unit/ProcessSessionTests.cpp

```diff
@@ -31,25 +32,16 @@ REGISTER_RESOURCE(DummyProcessor, "A processor that does nothing.");
 
 class Fixture {
  public:
-  Fixture();
   core::ProcessSession &processSession() { return *process_session_; }
 
  private:
   TestController test_controller_;
-  std::shared_ptr<TestPlan> test_plan_;
-  std::shared_ptr<core::Processor> dummy_processor_;
-  std::shared_ptr<core::ProcessContext> context_;
-  std::unique_ptr<core::ProcessSession> process_session_;
+  std::shared_ptr<TestPlan> test_plan_ = test_controller_.createPlan();
+  std::shared_ptr<core::Processor> dummy_processor_ = test_plan_->addProcessor("DummyProcessor", "dummyProcessor");
+  std::shared_ptr<core::ProcessContext> context_ = [this] { test_plan_->runNextProcessor(); return test_plan_->getCurrentContext(); }();
+  std::unique_ptr<core::ProcessSession> process_session_ = std::make_unique<core::ProcessSession>(context_);
 };
 
-Fixture::Fixture() {
```

Comment:
What's the motivation behind deleting this?

Reply:
Just wanted to simplify the code by moving the initialization code to default member initializers. I can revert if you have any objections.

Reply:
No, just didn't find the place in the diff where it was actually moved to. :) Got it, thanks!